README.Rmd

```diff
@@ -49,10 +49,12 @@ Sys.setenv(DUCKPLYR_OUTPUT_ORDER = TRUE)
 [![R-CMD-check](https://github.com/tidyverse/duckplyr/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/tidyverse/duckplyr/actions/workflows/R-CMD-check.yaml)
 <!-- badges: end -->
 
-The goal of the duckplyr R package is to provide a drop-in replacement for [dplyr](https://dplyr.tidyverse.org/) that uses [DuckDB](https://duckdb.org/) as a backend for fast operation.
-DuckDB is an in-process OLAP database management system, dplyr is the grammar of data manipulation in the tidyverse.
+> A **drop-in replacement for [dplyr](https://dplyr.tidyverse.org/)** that uses **[DuckDB](https://duckdb.org/)** as a backend for **fast operation**.
```

Comment:
I like the "quote with a tag line" approach that is seen in e.g. https://pak.r-lib.org/ or https://cli.r-lib.org/